analytics_dashboard/static/js/utils/globalization.js

```diff
@@ -1,5 +1,7 @@
 if (window.language === undefined) { // should only occur in test environments
     window.language = 'en';
+} else if (window.language === 'pt-BR') { // since there is no pt-BR file in https://github.com/unicode-cldr/cldr-numbers-modern/tree/master/main
```

Comment:
This is fine as a workaround for this specific ticket.

I can see 2 more standard solutions:
- add pt-br to cldr-numbers-modern as a Pull request
- instead of comparing to the `pt-BR` lang explicitly, make it convert all the languages to one of the compatible versions present in cldr-numbers-modern

Reply:
ufff, ya ví la lista de lo que si esta en `cldr-numbers-modern`! muchos perros, por que no pusieron pt-BR

Reply:
I read more in depth what the cldr is and I dont think that the pull request to cldr-numbers would be of any use. Maybe for the second option there is a standard tool. Something that checks the available languages and converts things of the form `la-LA` to `la` when needed